src/pandas_profiling/model/expectation_algorithms.py

```diff
@@ -0,0 +1,70 @@
+def generic_expectations(name, summary, batch, *args):
+    batch.expect_column_to_exist(name)
+
+    if summary["n_missing"] == 0:
+        batch.expect_column_values_to_not_be_null(name)
+
+    if summary["p_unique"] == 1.0:
+        batch.expect_column_values_to_be_unique(name)
+
+    return name, summary, batch
+
+
+def numeric_expectations(name, summary, batch, *args):
+    from great_expectations.profile.base import ProfilerTypeMapping
+
+    batch.expect_column_values_to_be_in_type_list(
+        name,
+        ProfilerTypeMapping.INT_TYPE_NAMES + ProfilerTypeMapping.FLOAT_TYPE_NAMES,
+        meta={
+            "notes": {
+                "format": "markdown",
+                "content": [
+                    "The column values should be stored in one of these types."
+                ],
+            }
+        },
+    )
+
+    if summary["monotonic_increase"]:
+        batch.expect_column_values_to_be_increasing(
+            name, strictly=summary["monotonic_increase_strict"]
+        )
+
+    if summary["monotonic_decrease"]:
+        batch.expect_column_values_to_be_decreasing(
+            name, strictly=summary["monotonic_decrease_strict"]
+        )
+
+    return name, summary, batch
+
+
+def categorical_expectations(name, summary, batch, *args):
+    return name, summary, batch
+
+
+def boolean_expectations(name, summary, batch, *args):
+    return name, summary, batch
+
+
+def path_expectations(name, summary, batch, *args):
+    return name, summary, batch
+
+
+def datetime_expectations(name, summary, batch, *args):

```

Comment:
This might be a good candidate for `expect_column_values_to_be_between` or at least `expect_column_min_to_be_between`. For `expect_column_values_to_be_between`, you can leave out the `max_value` to have a lower bound and no upper bound.